parlai/mturk/tasks/react_task_demo/react_custom_no_extra_deps/frontend/components/custom.jsx

```diff
@@ -38,19 +38,25 @@ class EvaluatorIdleResponse extends React.Component {
 class NumericResponse extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {'textval': ''};
+    this.state = {'textval': '', 'sending': false};
   }
 
   componentDidUpdate(prevProps, prevState, snapshot) {
+    // Only change in the active status of this component should cause a
```

Comment:
thanks for the commentary! this will make it easier for when people write their own custom components